modules/infinite-scroll/infinity.php

```diff
@@ -1033,7 +1035,9 @@ function action_wp_footer() {
 				extend( window.infiniteScroll.settings.scripts, <?php echo wp_json_encode( $scripts ); ?> );
 				extend( window.infiniteScroll.settings.styles, <?php echo wp_json_encode( $styles ); ?> );
 			})();
-		</script><?php
+		</script>
+		<span id="infinite-aria" aria-live="assertive"></span>
```

Comment:
In my testing making this `assertive` gave better results than `polite` - my only concern is on auto mode, the current flow of text being read will be interrupted. This makes sense for the click more button, not auto. Perhaps we change the value based on the setting?

Reply:
I agree with Enrique's feedback about using assertive for click mode & polite for auto 👍 

Also, I think you might only need this for the Loading state, since once the content loads & re-focuses on the new "page", it should announce that page label - you'll want to test that though, I don't have a jetpack dev site set up to try myself.

Reply:
Thanks, that makes sense to me also. 

>  you'll want to test that though, I don't have a jetpack dev site set up to try myself

I got mixed results in testing, I got variations of most things spoken at various points. At its best it went "loading", "loaded new posts", "page x".

Reply:
Now using `assertive` in click mode and `polite` in auto.